gluster-block.spec.in

```diff
@@ -52,6 +52,7 @@ rm -rf ${RPM_BUILD_ROOT}
 %{_sbindir}/gluster-block
 %{_sbindir}/gluster-blockd
 %{_mandir}/man8/gluster-block*.8*
+%dir %{_localstatedir}/log/gluster-block
```

Comment:
The problem was that the daemon did not start (due to iscsi daemon not being available). and hence the cli was segfaulting, since the dir did not exist (since the daemon had not created it)